test/Math/rlexe.xml

```diff
@@ -80,44 +26,14 @@
   </test>
   <test>
     <default>
-      <files>round.js</files>
-    </default>
-  </test>
-  <test>
-    <default>
-      <files>ceilfloor.js</files>
-      <baseline>ceilfloor.baseline</baseline>
-    </default>
-  </test>
-  <test>
-    <default>
-      <files>ceilfloor.js</files>
-      <baseline>ceilfloor.baseline</baseline>
-      <compile-flags>-sse:3</compile-flags>
-    </default>
-  </test>
-  <test>
-    <default>
-      <files>sin.js</files>
-      <baseline>sin.baseline</baseline>
-    </default>
-  </test>
-  <test>
-    <default>
-      <files>sqrt.js</files>
-      <baseline>sqrt.baseline</baseline>
-    </default>
-  </test>
-  <test>
-    <default>
-      <files>tan.js</files>
-      <baseline>tan.baseline</baseline>
+      <files>basics.js</files>
+      <compile-flags>-args summary -endargs</compile-flags>
     </default>
   </test>
   <test>
     <default>
-      <files>constants.js</files>
-      <baseline>constants.baseline</baseline>
+      <files>basics.js</files>
+      <compile-flags>-sse:3 -args summary -endargs</compile-flags>
```

Comment:
As one of the existing tests I've combined was being run both with and without sse:3 I've run the new combined test with and without that flag.